crates/client/tests/admin.rs

```diff
@@ -124,7 +123,6 @@ async fn storage_info() {
             roles_settings: None,
             source: AppBundleSource::Path(PathBuf::from("./fixture/test.happ")),
             ignore_genesis_failure: false,
```

Comment:
Was this monstrosity DPKI related or was this some CHC thing?

Reply:
Ah yes, I assume it's to do with CHC. I don't think it was introduced with DPKI, but I can't say for sure.

Reply:
Yes, introduced as part of CHC https://github.com/holochain/holochain/pull/2698/files#diff-fb2613144f7829236bd190ff6ec88919cc2475832d14580b77bf7f63cbf48702R10